platform/qt_location_service.cpp

```diff
@@ -130,7 +130,7 @@ void QtLocationService::Start()
 {
   if (!m_positionSource)
   {
-    LOG(LWARNING, ("No supported source is available for starting the positioning with:", m_positionSource->sourceName().toStdString()));
+    LOG(LWARNING, ("No supported source is available for starting the positioning with:", m_sourceName));
```

Comment:
I'd prefer to remove this log (and below). Just
```
if (m_positionSource)
{
  ...
}
```

The first one log in ctor is enough.

Reply:
I have removed the additional logs.